src/main/java/org/takes/facets/fallback/FbStatus.java

```diff
@@ -45,7 +47,46 @@
  */
 @EqualsAndHashCode(callSuper = true)
 public final class FbStatus extends FbWrap {
-
+    /**
+     * Ctor.
+     * @param code HTTP status code
+     * @since 0.16.10
+     */
+    public FbStatus(final int code) {
+        this(new Condition<Integer>() {
+            @Override
+            public boolean fits(final Integer status) {
+                return code == status.intValue();
+            }
+        });
+    }
+    /**
+     * Ctor.
+     * @param check HTTP status code predicate
+     * @since 0.16.10
+     */
+    public FbStatus(final Condition<Integer> check) {
+        this(check, new Fallback() {
+            @Override
+            public Iterator<Response> route(final RqFallback req)
+                throws IOException {
+                final Response res = new RsWithStatus(req.code());
+                final String message = res
```

Comment:
`message` looks like a one-time variable, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#one-time-variables